### ISSUE TITLE
mute when blackout is activated

### ISSUE THREAD

**Issue by omgbebebe** [alerta/alerta#1284]
Hi,

Is it possible to change the status of an already triggered alert when creating a new blackout. This can be useful when the operator has forgotten to activate a blackout before starting maintenance or in case of an automation failure.
In my case, I have a custom plugin that sends alert change notifications to a third party system, and in my opinion it would be nice to be able to mute (send a acknowledged status to external system) a bunch of alerts in case of human error or script failure.
I tried to solve this problem via plugins, but with no success, as as I see the blackout is just a filter, not an event (blackout_<created|changed|deleted>) that can be handled in plugins.

In short, my desired workflow is:
- Alerta already has fired alerts
- The operator creates a new blackout
- all alerts that match a blackout expression change status to 'blackout' or whatever else that makes sense
- all new alerts are processed as usual according to the parameters of the default blackout plugin
- (optional) if blackout was changed/removed, then all alerts that were affected by this blackout change their states according to the new blackout parameters.

Maybe you can point me to how this can be done with an existing event model?

**Comment by satterly** [comment]
If you want to trigger an action on an alert you can write a custom action like so https://github.com/alerta/alerta/pull/708

**Comment by satterly** [comment]
Closing due to inactivity. If you believe this issue has been closed prematurely please provide more information to help progress the issue. If you do not know what is expected you can ask on [Gitter chat](https://gitter.im/alerta/chat).
